ajax/dropdownTargets.php

```diff
@@ -1,29 +1,34 @@
 <?php
-/*
- -------------------------------------------------------------------------
- Deploy plugin for GLPI
- Copyright (C) 2022 by the Deploy Development Team.
 
- https://github.com/pluginsGLPI/deploy
- -------------------------------------------------------------------------
-
- LICENSE
-
- This file is part of Deploy.
-
- Deploy is free software; you can redistribute it and/or modify
- it under the terms of the GNU General Public License as published by
- the Free Software Foundation; either version 2 of the License, or
- (at your option) any later version.
-
- Deploy is distributed in the hope that it will be useful,
- but WITHOUT ANY WARRANTY; without even the implied warranty of
- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- GNU General Public License for more details.
-
- You should have received a copy of the GNU General Public License
- along with Deploy. If not, see <http://www.gnu.org/licenses/>.
- --------------------------------------------------------------------------
+/**
+ * -------------------------------------------------------------------------
+ * Deploy plugin for GLPI
+ * -------------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of Deploy.
+ *
+ * Deploy is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Deploy is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Deploy. If not, see <http://www.gnu.org/licenses/>.
+ * -------------------------------------------------------------------------
+ * @copyright Copyright (C) 2022-2023 by Deploy plugin team.
+ * @copyright Copyright (C) 2022 by the deploy plugin team.
+ * @license   GPLv3 https://www.gnu.org/licenses/gpl-3.0.html
+ * @license   MIT https://opensource.org/licenses/mit-license.php
+ * @link      https://github.com/pluginsGLPI/Deploy
+ * @link      https://github.com/pluginsGLPI/deploy
```

Comment:
Please run `./vendor/bin/licence-headers-check --discard-extra-tags` to remove duplicated header tags.